### PR TITLE
[update]Task6

### PR DESCRIPTION
**Pull request by UKI0x**
## point of uncertainty・What I've tried.
Please make sure that the problem is resolved after 3 seconds, but please check if this is correct.

I've added a setTimeout to the content of issue 5 so that it resolves after 3 seconds.
Also, I changed the word "prepend" to "insertAdjacentElement".

### CODE REVIEW

File: Task6/script.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+];
+
+const fetchData = new Promise((resolve) => {
+  setTimeout(() => {
+    resolve(data);
+  }, 3000);
+});
+
+fetchData.then((value) => {
+  value.forEach((list) => {
```
Comment (kenmori) [inline Task6/script.js]: IMO
```suggestion
fetchData.then((values) => {
  values.forEach((value) => {
```

or

```suggestion
fetchData.then((items) => {
  items.forEach((item) => {
```

as you like.

File: Task6/script.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const app = document.getElementById("js-app");
+const fragment = document.createDocumentFragment();
```
Comment (kenmori) [inline Task6/script.js]: IMO
Place it as close to the calling side as possible.

**Review by kenmori — APPROVED** [review]

File: Task6/script.js (hunk 2 above)
Comment (UKI0x) [reply Task6/script.js]: I placed getElement and Fragment, which were specified in the first line, right before forEach.
こちらで対応しました
[https://github.com/UKI0x/js-Hands-on/pull/6/commits/c2dab839bca4903bb1489e95968ff8ffb9f04a52](https://github.com/UKI0x/js-Hands-on/pull/6/commits/c2dab839bca4903bb1489e95968ff8ffb9f04a52)

File: Task6/script.js (hunk 1 above)
Comment (UKI0x) [reply Task6/script.js]: Thanks.
I understood that naming is easier to understand when looping if you write it in the plural first and then in the singular.

こちらで対応しました
[https://github.com/UKI0x/js-Hands-on/pull/6/commits/c2dab839bca4903bb1489e95968ff8ffb9f04a52](https://github.com/UKI0x/js-Hands-on/pull/6/commits/c2dab839bca4903bb1489e95968ff8ffb9f04a52)

**Pull request merged**